website/docs/tutorials/06-deploying/02-migrations/invoke.md

```diff
@@ -0,0 +1,94 @@
+# `invoke`
+
+```python
+def invoke(
+    contract_address: int,
+    function_name: str,
+    inputs: list[int] | dict[str, Any] | None = None,
+    *,
+    config: SignedCheatcodeConfig | None = None,
+):
```

Comment:
```suggestion
) -> None:
```

Reply:
I dont understand the need to explicitly specify none here